lib/llm/LLMProvider.ts

```diff
@@ -17,6 +19,8 @@ export class LLMProvider {
     "gpt-4o": "openai",
```

Comment:
might make sense to have a type called `ModelVendor: "openai" | "anthropic"` and edit `modelToProviderMap` to have type `{ [key in AvailableModel]: ModelVendor }`